joomla/joomla-cms
# Admin Status module

#### Summary of Changes

Remove the extra spaces before the links
#### Before " View site"
#### After "View site"
#### Testing Instructions

Apply the patch and check that there are no leading spaces on any of the links in the admin status module at the bottom
